upadate forbiddenapis to latest version as it not looks like runnig on jdk 11

right now:
```
[INFO] --- forbiddenapis:2.2:check (default) @ checkstyle-sonar-plugin ---
[WARNING] Bundled version of ASM cannot parse bytecode of java.lang.Object class; marking runtime as not suppported.
[WARNING] Your Java runtime (OpenJDK Runtime Environment 12+32) is not supported by the forbiddenapis MOJO. Please run the checks with a supported JDK!
[INFO] 
```

after update to latest version 2.7:
```
[INFO] --- forbiddenapis:2.7:testCheck (default) @ checkstyle-sonar-plugin ---
[INFO] Scanning for classes to check...
[INFO] Reading bundled API signatures: jdk-unsafe-11
[INFO] Reading bundled API signatures: jdk-deprecated-11
[INFO] Reading bundled API signatures: jdk-system-out
[INFO] Reading bundled API signatures: jdk-non-portable
[INFO] Loading classes to check...
[INFO] Scanning classes for violations...
[ERROR] Forbidden method invocation: java.lang.String#format(java.lang.String,java.lang.Object[]) [Uses default locale]
[ERROR]   in org.checkstyle.plugins.sonar.RunPluginTest (RunPluginTest.java:244)
[ERROR] Forbidden method invocation: java.lang.String#format(java.lang.String,java.lang.Object[]) [Uses default locale]
[ERROR]   in org.checkstyle.plugins.sonar.RunPluginTest (RunPluginTest.java:257)
[ERROR] Forbidden method invocation: java.lang.String#format(java.lang.String,java.lang.Object[]) [Uses default locale]
[ERROR]   in org.checkstyle.plugins.sonar.RunPluginTest (RunPluginTest.java:275)
[ERROR] Forbidden method invocation: java.lang.reflect.AccessibleObject#isAccessible() [Deprecated in Java 11]
[ERROR]   in org.sonar.plugins.checkstyle.CheckstyleConstantsTest (CheckstyleConstantsTest.java:34)
[ERROR] Forbidden method invocation: java.lang.reflect.AccessibleObject#isAccessible() [Deprecated in Java 11]
[ERROR]   in org.sonar.plugins.checkstyle.CheckstyleSeverityUtilsTest (CheckstyleSeverityUtilsTest.java:74)
[ERROR] Scanned 21 class file(s) for forbidden API invocations (in 0.11s), 5 error(s).
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

```

--- a/pom.xml
+++ b/pom.xml
@@ -497,7 +497,7 @@
       <plugin>
         <groupId>de.thetaphi</groupId>
         <artifactId>forbiddenapis</artifactId>
-        <version>2.2</version>
+        <version>2.7</version>
         <configuration>
           <targetVersion>${java.version}</targetVersion>
           <failOnUnsupportedJava>false</failOnUnsupportedJava>

--- a/src/it/java/org/checkstyle/plugins/sonar/RunPluginTest.java
+++ b/src/it/java/org/checkstyle/plugins/sonar/RunPluginTest.java
@@ -27,6 +27,7 @@ import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.List;
+import java.util.Locale;
 import java.util.Map;
 import java.util.stream.Collectors;
 
@@ -241,7 +242,8 @@ public class RunPluginTest {
                     .setParam("repositories", "checkstyle")
                     .executeUnsafely();
             if (!activateRulesResponse.isSuccessful()) {
-                fail(String.format("Failed to activate all rules. %s",
+                fail(String.format(Locale.ROOT,
+                        "Failed to activate all rules. %s",
                         activateRulesResponse.getBodyAsString()));
             }
             // deactivate some rules for test project
@@ -254,7 +256,8 @@ public class RunPluginTest {
                         .setParam("profile_key", profileKey)
                         .executeUnsafely();
                 if (!deactivateRulesResponse.isSuccessful()) {
-                    fail(String.format("Failed to deactivate rule %s. %s",
+                    fail(String.format(Locale.ROOT,
+                            "Failed to deactivate rule %s. %s",
                             ruleKey,
                             deactivateRulesResponse.getBodyAsString()));
                 }
@@ -272,7 +275,8 @@ public class RunPluginTest {
                 .setParam("projectKey", PROJECT_KEY)
                 .executeUnsafely();
         if (!assignQpResponse.isSuccessful()) {
-            fail(String.format("Failed to add project to quality profile. %s",
+            fail(String.format(Locale.ROOT,
+                    "Failed to add project to quality profile. %s",
                     assignQpResponse.getBodyAsString()));
         }
 

--- a/src/test/java/org/sonar/plugins/checkstyle/CheckstyleConstantsTest.java
+++ b/src/test/java/org/sonar/plugins/checkstyle/CheckstyleConstantsTest.java
@@ -31,7 +31,7 @@ public class CheckstyleConstantsTest {
     public void privateConstructor() throws ReflectiveOperationException {
         final Constructor<CheckstyleConstants> constructor = CheckstyleConstants.class
                 .getDeclaredConstructor();
-        assertThat(constructor.isAccessible()).isFalse();
+        assertThat(constructor.canAccess(null)).isFalse();
         constructor.setAccessible(true);
         constructor.newInstance();
     }

--- a/src/test/java/org/sonar/plugins/checkstyle/CheckstyleSeverityUtilsTest.java
+++ b/src/test/java/org/sonar/plugins/checkstyle/CheckstyleSeverityUtilsTest.java
@@ -71,7 +71,7 @@ public class CheckstyleSeverityUtilsTest {
     public void privateConstructor() throws ReflectiveOperationException {
         final Constructor<CheckstyleSeverityUtils> constructor = CheckstyleSeverityUtils.class
                 .getDeclaredConstructor();
-        assertThat(constructor.isAccessible()).isFalse();
+        assertThat(constructor.canAccess(null)).isFalse();
         constructor.setAccessible(true);
         constructor.newInstance();
     }

